docs/main_text.md

```diff
@@ -362,20 +362,31 @@ Experiment metadata (e.g., experiment description and title) is delivered in the
 
 # Species compendia
 
-refine.bio is currently in beta.
-Once refine.bio reaches production, we will periodically release compendia comprised of all the samples from a species that we were able to process.
+We periodically release compendia comprised of all the samples from a species that we were able to process.
 We refer to these as **species compendia**.
-We'll process these compendia in a manner that is different from the options that are available via the web user interface.
-We describe our intended processing pipeline below.
-Instead of selecting only genes available in all samples, we take the union of all genes, filling in any missing values with `NA` (e.g., perform a full outer join as illustrated below).
+We process these compendia in a manner that is different from the options that are available via the web user interface.
+These species compendia provide a snapshot of the most complete collection of gene expression that refine.bio can produce for each supported organism.
+
+The refine.bio web interface does an inner join when datasets are combined, so only genes present in all datasets are included in the final matrix.
+For compendia, we take the union of all genes, filling in any missing values with `NA`. 
+This is a "full outer join" as illustrated below. 
```

Comment:
Would it make sense to add rationale for choosing outer join? (It's not obvious to me but could be to researchers)

Reply:
I believe it is likely to be obvious to folks that are most interested in this offering, but that's a hunch based on my experience. I'm happy to change it if that turns out not to be the case.